### PR TITLE
Implemented random censors

### PR DESCRIPTION
**Pull request by Guzaboo**
Arco will now censor a new randomly selected word from the English language every day.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -105,9 +111,12 @@ client.on('message', message => {
     console.log('Checking: ' + messageSplit[i])
     console.log('Simplified: ' + simpleWord)
 
-    console.log('Has profanity: ' + censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, '')))
-
-    if (censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, ''))) {
+    console.log('Has profanity: ' + (censor.hasWord(simpleWord || notSimpleWord || englishWords.indexOf(simpleWord) != 1 && englishWords.indexOf(simpleWord) < d || englishWords.indexOf(notSimpleWord) != 1 && englishWords.indexOf(notSimpleWord) < d))
+    
+    let d = Math.round((new Date().getTime() - 1516492800000) / 86400000 + 1)
```
Comment (vpzomtrrfrt) [inline index.js]: duplicate code is bad

File: index.js [hunk 2]
```diff
@@ -68,13 +70,17 @@ client.on('ready', () => {
   console.log('Located IP channel: ' + ipChannel.name)
 
   checkIPChange(publicIP)
+  
+  let d = Math.round((new Date().getTime() - 1516492800000) / 86400000 + 1)
```
Comment (vpzomtrrfrt) [inline index.js]: magic numbers are bad

File: index.js (hunk 1 above)
Comment (Guzaboo) [reply index.js]: I was just mirroring the format that @LEARAX was using ¯\_(ツ)_/¯

File: index.js [hunk 3]
```diff
@@ -105,9 +111,12 @@ client.on('message', message => {
     console.log('Checking: ' + messageSplit[i])
     console.log('Simplified: ' + simpleWord)
 
-    console.log('Has profanity: ' + censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, '')))
-
-    if (censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, ''))) {
+    console.log('Has profanity: ' + (censor.hasWord(simpleWord || notSimpleWord || englishWords.indexOf(simpleWord) != 1 && englishWords.indexOf(simpleWord) < d || englishWords.indexOf(notSimpleWord) != 1 && englishWords.indexOf(notSimpleWord) < d))
```
Comment (vpzomtrrfrt) [inline index.js]: wouldn't this make all words profanity?

File: index.js [hunk 4]
```diff
@@ -68,13 +70,17 @@ client.on('ready', () => {
...
+  let d = Math.round((new Date().getTime() - 1516492800000) / 86400000 + 1)
+  console.log("Currently censoring " + d + "random words.");
 })
 
 client.on('disconnect', event => {
   console.log('!Disconnected: ' + event.reason + ' (' + event.code + ')!')
 })
 
 
+
```
Comment (vpzomtrrfrt) [inline index.js]: irrelevant change

File: index.js (hunk 1 above)
Comment (vpzomtrrfrt) [reply index.js]: where is the precedent for this?

Comment (Guzaboo) [reply index.js]: In the original code in @LEARAX's repository

File: index.js (hunk 2 above)
Comment (Guzaboo) [reply index.js]: you are bad

File: index.js (hunk 4 above)
Comment (Guzaboo) [reply index.js]: blame github

**Pull request closed without merging**